src/tools/genpolicy/src/utils.rs

```diff
@@ -22,6 +22,13 @@ struct CommandLineOptions {
     )]
     config_map_file: Option<String>,
 
+    #[clap(
+        long,
+        num_args = 0..=std::usize::MAX,
+        help = "Optional Kubernetes YAML input file paths"
+    )]
+    data_source_files: Option<Vec<String>>,
```

Comment:
@3u13r , I would strongly prefer to use here:

--config-file

Reasons:

1. This continues to work:

genpolicy -c cm1.yaml -y pod.yaml

2. -c is used by genpolicy for Config Map and/or Secret files. K8s calls those "Config and Storage Resources", so --config-file seems like a reasonable enough command line argument name.

3, AKS-CC users are already using --config-file.

Reply:
Sounds reasonable to me, changed it to `--config-file`.